manifests/config.pp

```diff
@@ -80,6 +80,17 @@
     ensure  => absent,
   }
 
+  if $::foreman::jobs_sidekiq_redis_url != undef {
```

Comment:
Should we wrap this whole block in `if $::foreman::jobs_manage_service {` like is done in `install.pp`?

Reply:
probably, yeah it would be more straight forward.